sys-devel/gcc/gcc-5.4.0_2016_12_12.recipe

```diff
@@ -0,0 +1,405 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
+CHECKSUM_SHA256="608df76dec2d34de6558249d8af4cbee21eceddbcb580d666f7a5a583ca3303a"
+REVISION="4"
```

Comment:
Anything for love
Oh, I would do anything for love
I would do anything for love, but I won't do that
No, I won't do that